### PR TITLE
Util `fromArcToBeziers()` API update

### PR DESCRIPTION
**Pull request by jasonsturges**
Proposing API change:
- Currently: `fromArcToBeizers`
- Proposing: `fromArcToBeziers`

### CODE REVIEW

**Comment by asturur** [comment]
i have to check how old is that util, in case we can make a correct alias, and deprecate the old one, but not remove it right away.

**Comment by jasonsturges** [comment]
@asturur Good idea, sorry - hadn't thought of that.

Looks like it was added August 20, 2020
SHA: 1c2164bab523da2405bbafb395d31c2b0584fc33
Pull Request: https://github.com/fabricjs/fabric.js/pull/6504

Would you like me to add a deprecation?  

Sorry for the hassle - certainly no major issue here, if you'd like to close and consider this some other time.

**Comment by asturur** [comment]
you can correct the function name and export, then add an alias with the wrong old name and a @deprecated notic in the jsdoc, with a line explaining that was a typo

File: src/util/path.js [hunk 1]
```diff
@@ -692,16 +692,16 @@
...
+    beziers.forEach(function(bezier) {
+      ctx.bezierCurveTo.apply(ctx, bezier.slice(1));
     });
   };
 
   fabric.util.parsePath = parsePath;
   fabric.util.makePathSimpler = makePathSimpler;
   fabric.util.getPathSegmentsInfo = getPathSegmentsInfo;
-  fabric.util.fromArcToBeizers = fromArcToBeizers;
+  fabric.util.fromArcToBeziers = fromArcToBeziers;
```
Comment (asturur) [inline src/util/path.js:704]:
```suggestion
  fabric.util.fromArcToBeziers = fromArcToBeziers;
    /**
   * Typo of `fromArcToBeziers` kept for not breaking the api once corrected.
   * Will be removed in fabric 5.0
   * @deprecated
   */
  fabric.util. fromArcToBeizers = fromArcToBeziers;
```

Comment (asturur) [reply src/util/path.js:704]: I wonder if this suggestion from one line to many can just work.

**Comment by jasonsturges** [comment]
@asturur Faster than me - just beat me to it.

I had added:

image

Which I think is working as intended:

image

But I won't push, as it looks like we did the same thing.

**Comment by jasonsturges** [comment]
Removed some whitespace and had to revert the change I made (sorry, I had pushed at the same time).

I think this is good now if you want to review the final pull request.

**Comment by asturur** [comment]
I added the suggestion to save you typing!
Thanks for the PR merging now

**Review by asturur — APPROVED** [review]

**Pull request merged**